### ISSUE TITLE
Updating routes

### ISSUE THREAD

**Issue by lgalabru** [blockstack/stacks-blockchain-api#350]
What's our take on updating the API routes and go with something more REST compliant?
`/new_mempool_tx` -> `/mempool_tx/new`
`/new_block`-> `/block/new`
`/new_burn_block` -> -> `/burn_block/new`

I was also thinking about maybe prefixing these endpoints with `stacks` or something (so `stacks/blocks/new`, etc).
Thoughts?

**Comment by agraebe** [comment]
@zone117x is this still relevant?

**Comment by zone117x** [comment]
I agree with @lgalabru that the paths names could be improved, but pretty low priority

**Comment by agraebe** [comment]
this seems related to #744

**Comment by sabbyanandan** [comment]
It appears the overall API nouns/verbs, primitives, and naming conventions could benefit from an uplift. This could feed into major refactoring, though, and we have to carefully navigate, not breaking the existing APIs. That would be to keep the old APIs (but deprecate them) and open up the newly refreshed API endpoints. 

All that to say, it is not impacting the end-users or directly adds value yet. Agree with @zone117x to keep this on the back burner for now.
